.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: Apache-2.0
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"  # See docs, they say not to use '.github'
+    schedule:
+      interval: "weekly"
+    commit-message:
+      prefix: "[CI]"
```

Comment:
[consider] I notice in one [random example](https://birtony.medium.com/breaking-ci-with-dependabot-weekly-telescope-podcast-af9884214187) that there is a space after the prefix, which would make sense.

Reply:
See what you think in the screenshot above. It also insists on the `:`.... but its prob not tooo bad is it?

Reply:
Yeah that's fine with me, unfortunate about the `:`, maybe it would be clever if you added a space and so not auto-add a `: `?  But anyway, fine as it is.